shared-routes2/e2e/CheckApp2.cy.ts

```diff
@@ -0,0 +1,46 @@
+import {BaseMethods} from "../../cypress/common/base";
+import {baseSelectors} from "../../cypress/common/selectors";
+import {Constants} from "../../cypress/fixtures/constants";
+import * as constants from "constants";
+const basePage: BaseMethods = new BaseMethods()
+
+describe('Check Shared-routes App2 starts and running', () => {
```

Comment:
no need for separate files. As I see the only difference in checks are hosts. so merge it one commonChecks file and walk through hosts by forEach

Reply:
fixed